File: 1603-running-sum-of-1d-array/running-sum-of-1d-array.cpp
class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        vector<int>::iterator itr;
        vector<int> result;
        int ans=0;
        for(int i=0;i<nums.size();i++){
            
            ans=ans+nums.at(i);
            result.push_back(ans);

        }
        return result;
    }
};

File: 1603-running-sum-of-1d-array/running-sum-of-1d-array.java
class Solution {
    public int[] runningSum(int[] nums) {
       int[] res=new int[nums.length];
       int total=0;
       for(int i=0;i<nums.length;i++){
        total+=nums[i];
        res[i]=total;
       } 
       return res;
    }
}